deploy/operator/namespace.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    control-plane: controller-manager
```

Comment:
we should use full dns prefix for every label, but I am not sure we even use that